single_tech_samples/synapseanalytics/sample3_integration_testing/infrastructure/modules/synapse_sql_pool.bicep

```diff
@@ -0,0 +1,53 @@
+param project string
+param env string
+param location string = resourceGroup().location
+param deployment_id string
+
+param sql_server_username string = 'sqlAdmin'
+@secure()
+param sql_server_password string
+
+
+resource sql_server 'Microsoft.Sql/servers@2021-02-01-preview' = {
+  name: '${project}-sql-${env}-${deployment_id}'
+  location: location
+  tags: {
+    DisplayName: 'SQL Server'
+    Environment: env
+  }
+  properties: {
+    administratorLogin: sql_server_username
+    administratorLoginPassword: sql_server_password
+  }
+
+  resource synapse_dedicated_sql_pool 'databases@2021-02-01-preview' = {
+    name: '${project}-syndp-${env}-${deployment_id}'
+    location: location
+    tags: {
+      DisplayName: 'Synapse Dedicated SQL Pool'
+      Environment: env
+    }
+    sku: {
+      name: 'DW100c'
+      tier: 'DataWarehouse'
+    }
+    properties: {
+      collation: 'SQL_Latin1_General_CP1_CI_AS'
+    }
+  }
+
+  resource firewall_rules 'firewallRules@2021-02-01-preview' = {
+    name: 'AllowAllAzureIps'
+    properties: {
+      endIpAddress: '0.0.0.0'
+      startIpAddress: '0.0.0.0'
+    }
+  }
```

Comment:
This is a sample, so this is perfectly fine. But I think it's worthwhile to ensure the audience knows that the synapse that gets created is accessible via the public internet (therefore to avoid putting any real data).

You may already be doing this as part of the README or the CLI prompts, if so, please auto-resolve this comment :)

Reply:
I'm updating the README!